wct.conf.json

```diff
@@ -8,7 +8,7 @@
             "browsers": [{
                 "browserName": "microsoftedge",
                 "platform": "Windows 10",
-                "version": ""
+                "version": "16"
```

Comment:
See my comments at https://github.com/Starcounter/starcounter-include/pull/91#issuecomment-367852325  It seems tests /behavior is flaky, so we could live with the default setup, then investigate the failure.

Maybe as a separate issue, not to block the demo? @warpech

Reply:
I'm not sure if we need it now that you mentione flakyiness. I changed it to 16 and tests passed. I chose 16 because I tested manually on Edge 16 and they passed. I'll test again.

Reply:
I was restarting those 3 builds in Travis manually and 30% attempts pass. So I'd rever back to `                "version": ""`. Tham move forward, and try to address this flaky test later.

Reply:
@warpech agreed on this as well on Slack